docker-compose-mastodon-update.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+git stash
+git fetch --tags
+latestTag=$(git describe --tags `git rev-list --tags --max-count=1`)
+git checkout $latestTag
```

Comment:
This looks a lot better! One more thing: can you add some documentation in this script, or the documentation repo, that explains what this script is, what it does, how to use it, and the limitations? (i.e.: if someone has made commits on their fork, as I have, a `checkout` is going to remove them.) Looking great!